portfolio/src/main/webapp/style.css

```diff
@@ -1,23 +1,48 @@
+/* Define color scheme of page */
 :root {
     --primary: #ffcece;
     --primary-dark: #ffc1c8;
     --accent: #ffe3b0;
     --contrast: #8ed6ff;  
 }
-
+/* BACKGROUND IMAGES STYLES */
+#body {
+    background-position: center;
+    background-image: url(/images/background/sunset3.jpg);
+    background-attachment: fixed;
+    min-height: 100%;
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 100vh
+}
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
+/* PROFILE STYLES */
+#profile-pic {
+    border-radius: 50%;
+    height: 100px;
+}
+#profile{
```

Comment:
Use a space between the last selector and the declaration block.

See https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation